core/transfer/src/main/java/org/eclipse/dataspaceconnector/transfer/core/transfer/TransferProcessManagerImpl.java

```diff
@@ -85,7 +85,7 @@
  * If no processes need to be transitioned, the transfer manager will wait according to the the defined {@link TransferWaitStrategy} before conducting the next iteration.
  * A wait strategy may implement a backoff scheme.
  */
-public class TransferProcessManagerImpl implements TransferProcessManager {
+public class TransferProcessManagerImpl extends CommandQueueProcessor<TransferProcessCommand> implements TransferProcessManager {
```

Comment:
Personally I would prefer composition over abstraction especially on specific features like these. 
An internal field instantiated on declaration would do the job nicely, and `CommandQueueProcessor` could be not abstract.

Reply:
Hi, I refactored it. Could you check and see if this is that what you had in mind?